apps/percona/functional/snapshot_rebuild/run_litmus_test.yml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: litmus-k8s-snapshot-rebuild-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: k8s-snapshot-litmus-rebuild
+
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: default
+
+          - name: APP_LABEL
+            value: 'name=percona'
+
+            # Application pvc
+          - name: APP_PVC
+            value: percona-mysql-claim
+
+            # Application namespace
+          - name: APP_NAMESPACE
+            value: percona-cstor
+
+          - name: SNAPSHOT
```

Comment:
Can we autogenerate a unique name in the test_vars itself and use it - trying to understand the utility of the ENV here. If it is not something we want to tune OR pass on to other related jobs -- we might as well keep it as a local test_var. Pretty sure we will be introducing more 'env tunables' into these tests going ahead -- so lets not convert into an ENV unless really necessary...? But I could be wrong - and there is some background already?